rocksdb_admin/admin_handler.cpp

```diff
@@ -911,38 +914,32 @@ void AdminHandler::async_tm_backupDBToS3(
     std::unique_ptr<apache::thrift::HandlerCallback<std::unique_ptr<
       BackupDBToS3Response>>> callback,
     std::unique_ptr<BackupDBToS3Request> request) {
+  LOG(INFO) << "S3 Backup " << request->db_name << " to " << request->s3_backup_dir;
+
   AdminException e;
   const auto n = num_current_s3_sst_uploadings_.fetch_add(1);
   SCOPE_EXIT {
     num_current_s3_sst_uploadings_.fetch_sub(1);
   };
-
-  if (n >= FLAGS_max_s3_sst_loading_concurrency) {
-    auto err_str =
-        folly::stringPrintf("Concurrent uploading/downloading limit hits %d by %s",
-                            n, request->db_name.c_str());
-    SetException(err_str, AdminErrorCode::DB_ADMIN_ERROR, &callback);
-    LOG(ERROR) << err_str;
-    common::Stats::get()->Incr(kS3BackupFailure);
-    return;
-  }
-
   common::Timer timer(kS3BackupMs);
-  LOG(INFO) << "S3 Backup " << request->db_name << " to " << request->s3_backup_dir;
```

Comment:
what is reason to remove this log? this is a log message to inform, it is about to backup the whole db, seem ok to remain